api/grpc/server/server_linux.go

```diff
@@ -25,12 +25,18 @@ func (s *apiServer) AddProcess(ctx context.Context, r *types.AddProcessRequest)
 		GID:            r.User.Gid,
 		AdditionalGids: r.User.AdditionalGids,
 	}
-	process.Capabilities = r.Capabilities
+	// for backwards compat in the API set eibp
+	process.Capabilities = &ocs.LinuxCapabilities{
```

Comment:
LGTM

(currently thinking of a separate security-opt in Docker for ambient, will do later)